openedx/adg/lms/webinars/helpers.py

```diff
@@ -277,23 +277,32 @@ def send_webinar_invitation_emails(webinar, old_webinar_data=None):
     if invite_email_recipients:
         send_webinar_emails(MandrillClient.WEBINAR_CREATED, webinar, list(invite_email_recipients))
 
-    if unregistered_members_emails:
-        send_webinar_emails(MandrillClient.WEBINAR_TEAM_INVITE, webinar, list(unregistered_members_emails))
+    member_email_recipients_with_no_platform_account = unregistered_members_emails.intersection(updated_webinar_emails)
```

Comment:
Can we use `&` for set intersection to make it consistent with the other code. We are using `-` for set difference.

Reply:
I think `intersection` is more clear and readable as compared to `&`. WDYT?

Reply:
Then let's use `difference` instead of `-` to make it consistent everywhere.